pkg/products/threescale/reconciler.go

```diff
@@ -1842,22 +1881,100 @@ func (r *Reconciler) RolloutDeployment(ctx context.Context, name string) error {
 	return err
 }
 
-func (r *Reconciler) getUserDiff(kcUsers []keycloak.KeycloakAPIUser, tsUsers []*User) ([]keycloak.KeycloakAPIUser, []*User) {
+func (r *Reconciler) getUserDiff(ctx context.Context, serverClient k8sclient.Client, kcUsers []keycloak.KeycloakAPIUser, tsUsers []*User) ([]keycloak.KeycloakAPIUser, []*User, []*User) {
 	var added []keycloak.KeycloakAPIUser
+	var deleted []*User
+	var updated []*User
+
+	rhssoConfig, err := r.ConfigManager.ReadRHSSO()
+	if err != nil {
+		r.log.Warning("Failed to get rhsso config: " + err.Error())
+		return added, deleted, updated
+
+	}
```

Comment:
```suggestion
	if err != nil {
		r.log.Warning("Failed to get rhsso config: " + err.Error())
		return added, deleted, updated
	}
```
Empty line after return intentional ? :thinking: